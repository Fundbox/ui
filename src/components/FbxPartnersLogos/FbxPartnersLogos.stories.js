import { storiesOf } from '@storybook/vue'
import { withInfo } from 'storybook-addon-vue-info'

import FbxPartnersLogos from './FbxPartnersLogos.vue'
import summary from './FbxPartnersLogos.md'

const stories = storiesOf('Components/PartnersLogos', module)

stories.add('default', withInfo({ summary })(() => ({
  components: { FbxPartnersLogos },
  template: `<fbx-partners-logos></fbx-partners-logos>`
})))
