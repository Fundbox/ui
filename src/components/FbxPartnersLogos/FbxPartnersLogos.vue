<template>
  <div class="container">
    <div class="logo-container">
      <img class="logo" :src="PaypalLogo">
    </div>
    <div class="logo-container">
      <img class="logo" :src="IntuitLogo">
    </div>
    <div class="logo-container">
      <img class="logo" :src="SynchronyLogo">
    </div>
  </div>
</template>

<script>
import IntuitLogo from './logos/Intuit.svg'
import PaypalLogo from './logos/Paypal.svg'
import SynchronyLogo from './logos/Synchrony.svg'

export default {
  name: 'FbxPartnersLogos',
  data() {
    return { IntuitLogo, PaypalLogo, SynchronyLogo }
  },
}
</script>

<style lang="scss" scoped>
  .container {
    display: flex;
  }

  .logo-container {
    flex: 33.33%;

    img {
      width: 100%;
    }
  }


</style>
