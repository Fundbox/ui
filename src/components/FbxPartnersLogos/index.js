import FbxPartnersLogos from './FbxPartnersLogos.vue'
import { registerComponents, vueUse } from '../../utils/plugins'

const components = {
  FbxPartnersLogos
}

const VuePlugin = {
  install(Vue) {
    registerComponents(Vue, components)
  }
}

vueUse(VuePlugin)

export default VuePlugin
