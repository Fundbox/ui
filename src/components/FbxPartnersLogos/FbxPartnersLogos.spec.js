import { shallowMount } from '@vue/test-utils'
import FbxPartnersLogos from './FbxPartnersLogos.vue'

describe('Components/FbxPartnersLogos', () => {
  describe('snapshots', () => {
    it('renders the default correctly', () => {
      const wrapper = shallowMount(FbxPartnersLogos, {
        // mounting options https://vue-test-utils.vuejs.org/api/options.html
      })
      expect(wrapper.html()).toMatchSnapshot()
    })
  })
})
