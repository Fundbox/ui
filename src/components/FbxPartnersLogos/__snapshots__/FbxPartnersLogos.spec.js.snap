// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components/FbxPartnersLogos snapshots renders the default correctly 1`] = `
<div class="partners-logos">
  <div class="logo-container"><img src="test-file-stub" class="logo"></div>
  <div class="logo-container"><img src="test-file-stub" class="logo"></div>
  <div class="logo-container"><img src="test-file-stub" class="logo"></div>
</div>
`;
